addons/binding/org.openhab.binding.wifiled/src/main/java/org/openhab/binding/wifiled/handler/LEDStateDTO.java

```diff
@@ -149,9 +154,9 @@ public LEDStateDTO withIncrementedProgramSpeed(int step) {
     }
 
     public Color getColor() {
-        float hue = (float) this.getHue().intValue() / 360;
-        float saturation = (float) this.getSaturation().doubleValue() / 100;
-        float brightness = (float) this.getBrightness().intValue() / 100;
+        float hue = (float) this.getHue().doubleValue() / 360.0f;
```

Comment:
Would the preference way to do 360d or 360.0 ?

I am planning to change it do:
```
        float hue = (float) (this.getHue().doubleValue() / 360.0);
        float saturation = (float) (this.getSaturation().doubleValue() / 100.0);
        float brightness = (float) (this.getBrightness().doubleValue() / 100.0);
```

I will add some unit tests around these calculations. The issues I found was that when I for example input this: 36,6,0 (RGB) I could get back 36,4,0 die to integer rounding.